### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by ylt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,6 +24,95 @@ Or install it yourself as:
 
 TODO: Write usage instructions here
```
Comment (henryaddison) [inline README.md]: You can probably remove this unless you're planning on writing anything more

File: README.md [hunk 2]
```diff
@@ -24,6 +24,95 @@ Or install it yourself as:
...
+    OpenActive::Rpde::RpdeItem.new(
+        Id: "2",
+        Modified: 4,
+        State: OpenActive::Rpde::RpdeState::Deleted,
+        Kind: OpenActive::Rpde::RpdeKind::SessionSeries,
+        Data: nil
+    )
+]
+
+json_Ld = OpenActive::Rpde::RpdeBody.create_from_next_change_number("https://www.example.com/feed", 2, items).to_json
```
Comment (henryaddison) [inline README.md]:
```suggestion
json_ld = OpenActive::Rpde::RpdeBody.create_from_next_change_number("https://www.example.com/feed", 2, items).to_json
```

**Review by henryaddison — APPROVED** [review]
Looks good, no major problems (remove a TODO from README given you've now done it!).

**Review by henryaddison — CHANGES_REQUESTED** [review]
Missing documentation on how to deserialize a model instance from JSON

File: README.md (hunk 2 above)
Comment (ylt) [reply README.md]: Good spot

**Review by henryaddison — APPROVED** [review]

**Pull request merged**